checks/evaluation/maintained.go

```diff
@@ -59,17 +60,22 @@ func Maintained(name string,
 		return checker.CreateMinScoreResult(name, "project was created in last 90 days. please review its contents carefully")
 	}
 
-	commitsWithinThreshold := 0
-	numberOfIssuesUpdatedWithinThreshold := 0
-
+	var commitsWithinThreshold, numberOfIssuesUpdatedWithinThreshold int
+	var err error
 	for i := range findings {
 		f := &findings[i]
 		if f.Outcome == finding.OutcomePositive {
 			switch f.Probe {
 			case issueActivityByProjectMember.Probe:
-				numberOfIssuesUpdatedWithinThreshold = f.Values[issueActivityByProjectMember.NoOfIssuesValue]
+				numberOfIssuesUpdatedWithinThreshold, err = strconv.Atoi(f.Values[issueActivityByProjectMember.NoOfIssuesKey])
+				if err != nil {
+					return checker.CreateRuntimeErrorResult(name, sce.WithMessage(sce.ErrScorecardInternal, err.Error()))
+				}
 			case hasRecentCommits.Probe:
-				commitsWithinThreshold = f.Values[hasRecentCommits.CommitsValue]
+				commitsWithinThreshold, err = strconv.Atoi(f.Values[hasRecentCommits.CommitsValue])
```

Comment:
For consistency (and since it's being used as a key to a map), should this be something like `hasRecentCommits.NoOfCommitsKey`?

Reply:
3986739a733bc84c129704779b949b0f02192eb3